### ISSUE TITLE
https://booknet.com/

### ISSUE THREAD

**Issue by dipu-bd** [dipu-bd/lightnovel-crawler#430]

**Comment by SirGryphin** [comment]
This is the same site as https://litnet.com, which already has a source but  I'm getting this error on all sites:
```
requests.exceptions.SSLError: HTTPSConnectionPool(host='readernet.com', port=443): Max retries exceeded with url: /en/reader/bound-by-vows-b341096?c=3450972 (Caused by 
SSLError(SSLError(1, '[SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1123)')))
```

**Comment by dipu-bd** [comment]
these SSLErrors are probably coming from cloudflare. make sure that your `cloudscraper` package is up to date.

**Comment by SirGryphin** [comment]
Already made sure everything is up to date.
`Requirement already satisfied: cloudscraper>=1.2.58`

I've managed to get rid of that error and get it working, some of the code was trying to redirect to wrong domain for chapters. I've tested it and all is working now. I've tried getting the search working but can't figure it out.

Added to PR #875

**Comment by dipu-bd** [comment]
which part of the code was trying to redirect to wrong url? Do you mean this one: 

`abs_url = self.last_visited_url.replace('book', 'reader')`

**Comment by SirGryphin** [comment]
Yeah, it was meant to replace the word book with reader at the end of the url so it can get the chapters like this:
```
From: https://booknet.com/en/book/bound-by-vows-b341096
To: https://booknet.com/en/reader/bound-by-vows-b341096
```
Instead it was also replace the "book" in the "booknet.com" domain name which was linking to a nonexistent site. So I changed the code to this instead.

`abs_url = self.last_visited_url.replace('/en/book/', '/en/reader/')`
